src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -1965,6 +1967,19 @@ public IValue evaluate(OperatorBase.SafeVariablesGetter variables) throws Evalua
                     itemStack -> !itemStack.isEmpty() && itemStack.hasTag()
             )).build());
 
+    /**
+     * Get the tooltip lines of an itemstack.
+     */
+    public static final IOperator OBJECT_ITEMSTACK_TOOLTIP_LINES = REGISTRY.register(OperatorBuilders.ITEMSTACK_1_SUFFIX_LONG
+            .output(ValueTypes.LIST).symbol("tooltip_lines").operatorName("tooltiplines").interactName("tooltipLines")
```

Comment:
yea, I was basing it on the ItemStack method name, but makes sense for it to be short right now, since there's no version that returns it as a single object (maybe at some point after #1402 ?)